algrtm/src/components/DevelopSection.js

```diff
@@ -5,7 +5,7 @@ const DevelopSection = () => {
   return (
     <section className="flex flex-col gap-6 justify-center items-center text-center max-w-[21.4375rem] mt-[2.75rem] mx-[1rem] md:max-w-[54.875rem] md:mt-[7.75rem md:mx-[4.56rem] xl-mx-[17.56rem] bg-neutralLight-neutral100 dark:bg-neutralDark-neutral100">
       <div className="h2 md:h1 text-neutralLight-neutral10 dark:text-neutralDark-neutral10 leading-[6.02rem]">
-        We develop algorithmic solutions
+        Make Data Science Great Again
```

Comment:
Probably this text should be wrapped by h2 element and not by div to make it more semantic. The section should also have the margin set to 0 on small devices, because the parent div uses padding (so now with the margin it has 1rem added twice)

![image](https://github.com/Nospoko/algrtm/assets/26410548/84984018-dc15-4144-80c0-de8e8b3e3eab)